app/controllers/transactions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class TransactionsController < ApplicationController
+  # GET /transactions or /transactions.json
+  def index
+    @category = Category.find(params[:category_id])
+    @transactions = @category.transactions
```

Comment:
- [ ] For a given category, the list of transactions is presented ordered by the most recent. It will be nice to fix this. You can achieve this by creating the method in your helper and then add it to the `index` action here.
something similar to this: `@transactions = @category.transactions.most_recent`

![image](https://user-images.githubusercontent.com/51296741/161384281-002e19cb-4e14-4024-9785-3e5b7c57cc7e.png)